src/core/toga/widgets/canvas.py

```diff
@@ -506,21 +424,21 @@ class Stroke(CanvasContextMixin):
 
     def __init__(self, color=BLACK, line_width=2.0):
         super().__init__()
-        self._color = parse_color(color)
+        self._color = None
```

Comment:
Is this double up needed? The path for the setter doesn't have any dependency on a pre-existing value of `_color` AFAICT.

Reply:
I was trying to avoid my linter giving instance variables defined outside __init__ warnings, but I agree in this case it is a little wonky because we want to use the setter on initialization of the instance. I went ahead and removed the extra line.